server/odc-service/src/test/java/com/oceanbase/odc/service/sqlcheck/MySQLCheckerTest.java

```diff
@@ -1362,6 +1362,107 @@ public void check_wrongPlExists_violationGenerated() {
         Assert.assertEquals(expect, actual);
     }
 
+    @Test
+    public void check_restrictSqlAffectedRows4MySQL_violationGenerated() {
```

Comment:
unit test should involve your code logic which is parsing the resultset of explain SQL.

Reply:
Got it. The mock process has been adjusted.